source/_posts/2020-11-12-android-300-release.markdown

```diff
@@ -0,0 +1,128 @@
+
+## Breaking Changes
+
+Since we [last spoke](https://www.home-assistant.io/blog/2020/09/12/android-240-release/) we mentioned that we were going to be breaking up some sensors and their attributes. As of 3.0.0 we have completed this task and no longer have non-static attributes as part of any sensor. If you are missing any data please check the Manage Sensors screen under App Configuration and enable the sensor you were previously using as an attribute.
+
+Another major breaking change in this release is that the widgets were refactored to align with the application architecture. Unfortunately this means that some existing widgets may disappear from your home screen and will need to be recreated. We apologize for this and are unable to bring back the existing widgets that were lost. If you forgot what data was there we do have that saved, more on that down [below](#other-enhancements).
+
+## Onboarding Improvements
+
+The first big change is that our onboarding screen has changed, we now ask the user to enter their preferred device name that gets attached to all entities that are created by the app. This was a source of frustration for some users as this meant that anytime they logged into the app they would then need to rename their entities to the name that they actually want. By default the device name is the device model but that is not meaningful to some users, especially if they have more than one device of the same model.
+
+
+<p class='img'>
+<img src='/images/blog/2020-11-05-android-300-release/onboarding.png' alt='Screenshot of new onboarding flow'></a>
+Screenshot of new onboarding flow.
+</p>
+
+
+## New Sensors
+
+We have several new sensors to welcome to the app, all of which are disabled by default. The first set of sensors were actually introduced in 2.5.0:
+
+- 4 Traffic Stat sensors - Sensors whose state represent the amount of data sent and received by the device. Mobile data may not be accurate, it depends on the data tat we get from the API.
+
+The following sensors are new in 3.0.0:
+
+- 4 Keyguard sensors - Sensors that represent various states from the Keyguard API, such as if a device is currently locked or has a password setup. These sensors will update during the periodic 15 minute interval.
+- Last Notification sensor - A very powerful sensor that requires a special permission you must grant to read all notifications posted on the device. All attributes of the notification are provided as attributes for the sensor. You can think of this sensor as a great way to integrate any app that posts a notification to your device allowing you to automate around it. Personally I have been using it to help integrate apps that don't integrate with Home Assistant Core a food delivery app for example, you can tell when your food has been delivered and automate around it. This sensor will update as soon as a notification is posted.
```

Comment:
```suggestion
- Last Notification sensor - A very powerful sensor that requires a special permission you must grant to read all notifications posted on the device. All attributes of the notification are provided as attributes for the sensor. You can think of this sensor as a great way to integrate any app that posts a notification to your device allowing you to automate around it. Personally I have been using it to integrate a food delivery app, to detect when my order will be delivered and automate around it. This sensor will update as soon as a notification is posted.
```

This feature is AWESOME! The example didn't quite seem to come across clearly to me, maybe this wording is better maybe I've got the wrong end of the stick?

Reply:
I think what you have reads better, I was sorta repeating what I said in the previous sentence :)